data/crud/fund_round_queries.py

```diff
@@ -49,3 +55,71 @@ def get_rounds_for_application_deadline_reminders() -> Sequence[Round]:
 def set_application_reminder_sent(round: Round) -> None:
     round.application_reminder_sent = True
     db.session.commit()
+
+
+def get_rounds_with_passed_deadline() -> Sequence[Round]:
+    """
+    Retrieve rounds that have passed their deadline but have not yet had an event
+    created for sending incomplete applications.
+    """
+    now = datetime.now()
+    one_month_ago = now - timedelta(days=30)
+
+    rounds_with_passed_deadline = (
+        db.session.query(Round).filter(Round.deadline < now, Round.deadline >= one_month_ago).all()
+    )
+
+    rounds_without_event = [
+        round
+        for round in rounds_with_passed_deadline
+        if not db.session.query(Event)
+        .filter(Event.round_id == round.id, Event.type == EventType.SEND_INCOMPLETE_APPLICATIONS)
+        .first()
+    ]
+
+    return rounds_without_event
+
+
+def get_passed_round_applications(round: Round) -> Sequence[Applications]:
+    """
```

Comment:
Could we re-use `get_applications_for_round_by_status()` function for this part?